Create README.md

FrancisJMR opened the pull request:
New Researchers ID & Access Claims landing page.  flow_of_claims.png still needs to be added.

FrancisJMR committed:
Create README.md

New Researchers ID & Access Claims landing page.  flow_of_claims.png still needs to be added.

cdvoisin commented on researcher_ids/README.md:
```diff
@@ -0,0 +1,89 @@
+The genomics community was therefore in need of:
+
+- a consistent definition of who a bona fide researcher is
+- ne or more identity providers that respect this definition and provide identities in the virtual world that travel with the researcher across various data sharing repositories
```
Replace "ne" with "one"?

FrancisJMR replied:
Fixed in new commit.

cdvoisin commented on researcher_ids/README.md:
```diff
@@ -0,0 +1,89 @@
+
+## Flow Of Claims
+
+![alt text](https://github.com/ga4gh-duri/flow_of_claims.png "Flow of Claims")
```
The PNG file is missing?

cdvoisin replied:
Also it may be worth putting image assets in the /assets/img/ directory.

FrancisJMR replied:
I've changed the path to /assets/img/  --- I'd don't have push access to the repository to add the image in that folder however. Can you try it? The image should be named "flow_of_claims.png"

cdvoisin reviewed (commented):
Thanks for putting this together. How about we land it with the current links and then update the links later to point to the github docs instead of the Google docs.

FrancisJMR committed:
Update README.md

Type "ne" to "one" fixed and changed path to "flow_of_claims.png" to /assets/img/

FrancisJMR committed:
Update README.md

Added changes to the Introduction from the google doc.

FrancisJMR committed:
Add files via upload

Added /assets/img/flow_of_claims.png for the researcher_ids/README.md reference

FrancisJMR commented:
Ok I've added the png file and incorporated the changes to the introduction from Tommi and I found in the Google Doc.  I cannot merge this back to master but it should be good to go.

cdvoisin commented on researcher_ids/README.md:
```diff
@@ -0,0 +1,90 @@
+The genomics community was therefore in need of:
+
+- a consistent definition of who a bona fide researcher is
+- one or more identity providers that respect this definition and provide secure digital identities that researchers can use online to access in the virtual world that travel with the researcher across various federated data sharing repositories around the world
```
Remove: "in the virtual world that travel with the researcher across"
Remove: "sharing" (after the words "federated data")
